recipe/activate-cross-r-base.sh

```diff
@@ -0,0 +1,11 @@
+if [[ "$CONDA_BUILD_STATE" != "test" && "$build_platform" != "$target_platform" ]]; then
+  export R=$BUILD_PREFIX/bin/R
+  export R_ARGS="--library=$PREFIX/lib/R/library --no-test-load"
+  echo "R_HOME=$PREFIX/lib/R"     > $BUILD_PREFIX/lib/R/etc/Makeconf
+  cat $PREFIX/lib/R/etc/Makeconf >> $BUILD_PREFIX/lib/R/etc/Makeconf
+fi
+
+if [[ "$CONDA_BUILD" == "" ]]; then
```

Comment:
We should first check this and then touch files. Then we won't create random things in `/`